diktat-common/src/main/kotlin/org/cqfn/diktat/common/config/rules/RulesConfigReader.kt

```diff
@@ -84,6 +84,10 @@ open class RulesConfigReader(override val classLoader: ClassLoader) : JsonResour
     }
 }
 
+open class TestAnchorsConfiguration(config: Map<String, String>) : RuleConfiguration(config) {
```

Comment:
Can we somehow make it an object, that can be created only once for an entire common config block? It could be a lazy property, for example. We should also move accesses to domainName to this common config.